src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -0,0 +1,303 @@
+              This field is required.
+            </span>
+          </div>
+        </div>
+
+        <!-- Use a dedicated pool -->
+        <div class="form-group">
+          <div class="col-sm-offset-3 col-sm-9">
+            <div class="checkbox checkbox-primary">
+              <input type="checkbox"
+                     id="useDataPool"
+                     name="useDataPool"
+                     formControlName="useDataPool"
+                     (change)="onUseDataPoolChange()">
+              <label i18n
+                     for="useDataPool">Use a dedicated data pool</label>
+            </div>
+          </div>
+        </div>
+
+        <!-- Data Pool -->
+        <div class="form-group"
+             [ngClass]="{'has-error': (formDir.submitted || rbdForm.controls.dataPool.dirty) && rbdForm.controls.dataPool.invalid}"
+             *ngIf="rbdForm.controls.useDataPool.value">
+          <label class="control-label col-sm-3"
+                 for="dataPool">
+            Data pool
+            <span class="required"></span>
+            <cd-helper html="Dedicated pool that stores the object-data of the RBD.">
+            </cd-helper>
+          </label>
+          <div class="col-sm-9">
+            <select id="dataPool"
+                    name="dataPool"
+                    class="form-control"
+                    formControlName="dataPool"
+                    (change)="onDataPoolChange($event.target.value)">
+              <option *ngIf="dataPools === null"
+                      [ngValue]="null">Loading...
+              </option>
+              <option *ngIf="dataPools !== null && dataPools.length === 0"
+                      [ngValue]="null">-- No data pools available --
+              </option>
+              <option *ngIf="dataPools !== null && dataPools.length > 0"
+                      [ngValue]="null">-- Select a data pool --
+              </option>
+              <option *ngFor="let dataPool of dataPools"
+                      [value]="dataPool.pool_name">{{ dataPool.pool_name }}</option>
+            </select>
+            <span i18n
+                  class="help-block"
+                  *ngIf="(formDir.submitted || rbdForm.controls.dataPool.dirty) && rbdForm.controls.dataPool.hasError('required')">
+              This field is required.
+            </span>
+          </div>
+        </div>
+
+        <!-- Size -->
+        <div class="form-group"
+             [ngClass]="{'has-error': (formDir.submitted || rbdForm.controls.size.dirty) && rbdForm.controls.size.invalid}">
+          <label translate
+                 class="control-label col-sm-3"
+                 for="size">Size
+            <span class="required"></span>
+          </label>
+          <div class="col-sm-9">
+            <input id="size"
+                   name="size"
+                   class="form-control"
+                   type="text"
+                   formControlName="size"
+                   placeholder="e.g., 10GiB"
```

Comment:
If I type `100` it get's auto completed to `100MiB` but the placeholder races the expectation that it should be `GiB` instead.

Reply:
Done